mempool/v0/clist_mempool_test.go

```diff
@@ -151,9 +151,9 @@ func TestReapMaxBytesMaxGas(t *testing.T) {
 		{20, 10, 10, 0},
 		{20, 28, 10, 1}, // account for overhead in Data{}
 		{20, 240, 5, 5},
-		{20, 240, -1, 10},
+		{20, 240, -1, 12},
```

Comment:
[Non Blocking][Question]
Why the increase?

Reply:
https://github.com/celestiaorg/celestia-core/pull/879#discussion_r1019586683

Reply:
yes, sorry, just saw the comments, we were writing at the same  time :D